integration/substrate/array_struct_mapping_storage.spec.ts

```diff
@@ -4,7 +4,7 @@ import { ContractPromise } from '@polkadot/api-contract';
 
 describe('Deploy array_struct_mapping_storage contract and test', () => {
     it('array_struct_mapping_storage', async function () {
-        this.timeout(200000);
+        this.timeout(1000000);
```

Comment:
Unfortunately I don't think this will fix it (but instead give the spurious runs more time until they finally fail). I got these sporadic timeouts on my localhost as well, and tried to adjust the timeouts already without success :confused:

Reply:
ok, I've removed this part from the PR